main.go

```diff
@@ -2,8 +2,27 @@ package main
 
 import (
 	"dynamic-buildkite-template/cmd"
+	"dynamic-buildkite-template/config"
+	"os"
+
+	log "github.com/sirupsen/logrus"
 )
 
+func init() {
+	// Log as JSON instead of the default ASCII formatter.
```

Comment:
not sure if you need these comments. the code is kind of self explanatory. @stephen-fox what do you think?

Reply:
I am keeping these comments for now as reference